research/gam/README.md

```diff
@@ -40,6 +40,20 @@ More details can be found in our
 [slides](https://drive.google.com/open?id=1tWEMoyrbLnzfSfTfYFi9eWgZWaPKF3Uu) or
 [poster](https://drive.google.com/file/d/1BZNR4B-xM41hdLLqx4mLsQ4KKJOhjgqV/view).
 
+## Updated Results
+A bug was discovered in the implementation of the GAM agreement regularization term after publication. We have fixed the bug (PR #82) and have rerun the affected experiments. Below are the updated results (note that the GAM* results are not affected). 
+
+Dataset  | Method      | Updated Accuracy (mean ± stderr)
+-------- | :---------: | :-------------: 
+Cora     | MLP + GAM   | 80.2 ± 0.31 
```

Comment:
Which rows do these correspond to in Table 1 of the paper (in other words, what's the hidden layer configuration)? That'll help the reader understand the difference(s). Optionally, you may also add a column here listing the old values from the paper.

Reply:
Thanks for the suggestion! I updated the readme with a new table similar to the one in the paper, comparing the new results with the competing methods. Markdown doesn't support nice table formatting options, so I added it as an image. See my latest commit.